kube-scheduler-healthcheck

```diff
@@ -18,7 +18,7 @@ endOfPodDef
 
 alias kubectl="timeout -t 30 kubectl --namespace \$TEST_POD_NAMESPACE"
  
-kubectl delete pod $TEST_POD_NAME || true
+kubectl delete pod $TEST_POD_NAME --grace-period=0 || true
```

Comment:
Pondering here. If the node it's on is dead (broken docker, node shut down etc), this still won't do it.

Maybe we should be doing something like:
```
delete pod --grace-period=0
sleep 1
delete pod --grace-period=0 --force
```

Reply:
should we `|| true` all of those?

Reply:
Ah, yes